compiler/dyno/test/uast/testBuildIDs.cpp

```diff
@@ -147,7 +147,7 @@ static  void test2() {
   Context context;
   Context* ctx = &context;
 
-  auto builder = Builder::build(ctx, "path/to/test.chpl");
+  auto builder = Builder::createForTopLevelModule(ctx, "path/to/test.chpl");
   Builder* b   = builder.get();
```

Comment:
I noticed for the parser pattern you changed from `parser.get()` to just `&parser`. Is there a reason not to do that same thing with the builder?

Reply:
The Parser object is small, but the Builder object is not small. I was thinking there might be some advantage to heap allocating it.